addons/interaction/functions/fnc_initLaptop.sqf

```diff
@@ -22,15 +22,13 @@ else
 
 if(!isDedicated) then
 {
-    // add the armaOS Menu to ACE3 Interaction Menu
-    private _armaOSAction = ["AE3_ArmaOSAction", localize "STR_AE3_ArmaOS_Config_ArmaOSDisplayName", "", {}, {true}] call ace_interact_menu_fnc_createAction;
-    [_laptop, 0, ["ACE_MainActions"], _armaOSAction] call ace_interact_menu_fnc_addActionToObject;
+    private _parentActionPath = _equipment getVariable ["AE3_parentActionPath", []];
```

Comment:
Why don't u use addParentAction instead?

Reply:
I don't have access to the `_name` variable. To achieve that I would need to pass additional parameters to this function. Because of the way the code flows, this variable is already set and can easily be used.